src/pages/cexs/index.tsx

```diff
@@ -294,6 +294,38 @@ export async function getStaticProps() {
 			}
 		})
 	)
+	
+	async function fetchTVL() {
+		const tvls = [];
+	  
+		for (const platform of cexData) {
+		  const response = await fetch(`https://api.llama.fi/tvl/${platform.slug}`);
```

Comment:
Seems to be some problem here , when running locally i cant get the TVL values
![image](https://user-images.githubusercontent.com/93603962/209676715-2acb58ed-64cf-497a-8059-0fc3c9115bd6.png)